ansible/files/telemetry.sh

```diff
@@ -48,6 +48,9 @@ fi
 PRESTO_CONFIG_SCRIPT=$(cat <<EOF
 while ! pgrep presto > /dev/null; do sleep 1; done
 
+# install presto plugins
+sudo -u presto aws s3 sync $TELEMETRY_CONF_BUCKET/plugins/ /usr/lib/presto/plugin/
```

Comment:
We don't have any at the moment, I'd be happy to help make that happen but I think that may be out of scope right now.  I'd love to get this merged so this isn't a blocker anymore and we can have a larger discussion later.